src/zenml/zen_stores/schemas/pipeline_run_schemas.py

```diff
@@ -298,7 +298,7 @@ def to_model(
         )
 
         if self.deployment is not None:
-            deployment = self.deployment.to_model()
+            deployment = self.deployment.to_model(include_metadata=True)
```

Comment:
Why is this needed?

Reply:
It just saves a hydration call that would happen in the next line when we access some attributes of the deployment

Reply:
I see, thanks.